WeakAuras/GenericTrigger.lua

```diff
@@ -670,10 +670,12 @@ local function RunTriggerFunc(allStates, data, id, triggernum, event, arg1, arg2
       else
         ok, returnValue = xpcall(data.triggerFunc, errorHandler, allStates, event, arg1, arg2, ...);
       end
-      if (ok and (returnValue or (returnValue ~= false and allStates.__changed))) then
+      if (ok and (returnValue or (returnValue ~= false and allStates.IsChanged and allStates:IsChanged()))) then
```

Comment:
I think you can just call IsChanged here.

Reply:
i was solving an error caused by the __index mistake